common/conf.hpp

```diff
@@ -92,9 +92,128 @@ class CmdlineOptionsIterator {
 	ArgumentsMode mode_ {ArgumentsMode::RejectBareArguments};
 };
 
+// NOTE - When updating this class - either adding or removing variables. Be
+// sure to update the transitive dependencies also.
+class Paths {
+private:
+	string PathConfDir = conf::GetEnv("MENDER_CONF_DIR", path::Join("/etc", "mender"));
+	string RootfsScriptsPath = path::Join(PathConfDir, "scripts");
+	string ConfFile = path::Join(PathConfDir, "mender.conf");
+
+	string PathDataDir = conf::GetEnv("MENDER_DATA_DIR", path::Join("/usr/share", "mender"));
+	string ModulesPath = path::Join(PathDataDir, "modules/v3");
+	string IdentityScript =
+		path::Join(PathDataDir, "identity", "mender-device-identity");
+
+	string DataStore =
+		conf::GetEnv("MENDER_DATASTORE_DIR", path::Join("/var/lib", "mender"));
+	string ArtScriptsPath = path::Join(DataStore, "scripts");
+	string ModulesWorkPath = path::Join(DataStore, "modules/v3");
+	string BootstrapArtifactFile = path::Join(DataStore, "bootstrap.mender");
+	string FallbackConfFile = path::Join(DataStore, "mender.conf");
+
+	string KeyFile = "mender-agent.pem";
```

Comment:
These should all end with `_` because they are private. But I'd be the first one to break the rule for the sake of better readability, so not really suggesting a change here. :grin:

Reply:
Right, they should all have `variable_name_style_` actually, I didn't even think about that.